ast/ast_test.go

```diff
@@ -8,21 +8,28 @@ func TestStatement(t *testing.T) {
 	Statement(&QueryStatement{}).isStatement()
 	Statement(&CreateDatabase{}).isStatement()
 	Statement(&CreateTable{}).isStatement()
-	Statement(&CreateView{}).isStatement()
-	Statement(&CreateIndex{}).isStatement()
-	Statement(&CreateSequence{}).isStatement()
-	Statement(&CreateRole{}).isStatement()
 	Statement(&AlterTable{}).isStatement()
-	Statement(&AlterIndex{}).isStatement()
 	Statement(&DropTable{}).isStatement()
+	Statement(&CreateIndex{}).isStatement()
```

Comment:
Is comments not needed in `ast_test.go`?

Reply:
No, for now. I'd like to remove `ast_test.go`. This is invaluable. What are we testing with this?

Reply:
I agree to remove.
Most interface implementation are already tested in `parse*()` in `parser.go`, and maintainance ordering in `ast_test.go` is waste of time.
May only valuable thing be missing `TestNode()` because it is possible to fail to implement methods of `Node`, and it can't be detected unless it is used as `Node`.